src/server.c

```diff
@@ -150,62 +116,16 @@ void
 sig_handler(int sig) {
     if (force_exit)
         exit(EXIT_FAILURE);
-
-    char sig_name[20];
-    get_sig_name(sig, sig_name);
-    lwsl_notice("received signal: %s (%d), exiting...\n", sig_name, sig);
+    lwsl_notice("received signal: %s (%d), exiting...\n", strsignal(sig), sig);
```

Comment:
Please note there is `#ifdef __linux__` and `#endif` around the `sys_signame` definition in `utils.c`, you can verify this list with the output of `kill -L` on linux.

It seems the list you given is from BSD, the `sys_signame` is already defined by libc on BSD systems, it won't be redefined by ttyd.